draft-private-access-tokens.md

```diff
@@ -205,6 +205,36 @@ this token demonstrates to the Origin that the Client meets its policies
 (since they were enforced by the Issuer before issuing this token), and then
 provides the services or content gated behind these policies.
 
+## User Interaction
+
+When used in contexts like websites, origin servers that challenge clients for
+Private Access Tokens need to consider how to optimize their interaction model
+to ensure a good user experience.
+
+Private Access Tokens are designed to be used without explicit user involvement.
+Since tokens are only valid for a single origin and in response to a specific
+challenge, there is no need for a user to manage a limited pool of tokens
+across origins. The information that is available to an origin upon token
+redemption is limited to the fact that this is a client that passed a
+Mediator's checks and has not exceeded the per-origin limit defined by an
+Issuer. Generally, if a user is willing to use Private Access Tokens with
+a particular origin (or all origins), there is no need for per-challenge
+user interaction.
+
+Since tokens are issued using a separate connection through a Mediator
+to an Issuer, the process of issuance can add user-perceivable latency.
+Origins SHOULD NOT block all rendering of a website on token authentication.
```

Comment:
```suggestion
Origins SHOULD NOT block useful work on token authentication.
```

Reply:
Trying to de-emphasize the web context, and focus more on generic contexts in which access tokens may be needed.